shopinvader/models/shopinvader_variant.py

```diff
@@ -302,20 +300,15 @@ def pick_1st_variant(prods):
             # NOTE: if the order is changed by adding `asc/desc` this can be broken
             # but it's very unlikely that the default order for product.product
             # will be changed.
-            try:
```

Comment:
wouldn't be easier to remove the code from the order?

Reply:
I don't think so, as the aim is to keep the natif order in odoo.
For example, for some customer the default code is generated automatically (with a specific pattern) so every variant are ordered in a same way.
I thinks it's better to keep odoo order so if you customize it, it will work without any extra code